server/server.go

```diff
@@ -229,23 +229,7 @@ func (s *Server) initClusterID() error {
 		s.clusterID, err = initOrGetClusterID(s.client, pdClusterIDPath)
 		return errors.Trace(err)
 	}
-
-	key := string(resp.Kvs[0].Key)
-
-	// If the key is "pdClusterIDPath", parse the cluster ID from it.
-	if key == pdClusterIDPath {
-		s.clusterID, err = bytesToUint64(resp.Kvs[0].Value)
-		return errors.Trace(err)
-	}
-
-	// Parse the cluster ID from any other keys for compatibility.
-	elems := strings.Split(key, "/")
-	if len(elems) < 3 {
-		return errors.Errorf("invalid cluster key %v", key)
-	}
-	s.clusterID, err = strconv.ParseUint(elems[2], 10, 64)
-
-	log.Infof("init and load cluster id: %d", s.clusterID)
+	s.clusterID, err = bytesToUint64(resp.Kvs[0].Value)
```

Comment:
do we still need to log cluster ID here?

Reply:
it's already logged in another place.